### ISSUE TITLE
lddtool json export seems to export the whole IM, not just the current dictionary

### ISSUE THREAD

**Issue by msbentley** [NASA-PDS/pds4-information-model#723]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I add the J flag to lddtool, a large json is generated which seems to contain much more than the output of the current dictionary:

![image](https://github.com/NASA-PDS/pds4-information-model/assets/1671065/99dc04dd-4ae3-457b-9a9f-340cac6aa69b)

### 🕵️ Expected behavior

I expected the -J flag to putput the json for the current dictionary.

### 📜 To Reproduce

```bash
1.
2.
3.
...
```

### 🖥 Environment Info

```bash
MacOS 14.1.1
```

### 📚 Version of Software Used

LDDTool Version: 14.3.0-SNAPSHOT
Built with IM Version: 1.21.0.0
Build Date: 2023-08-23 23:18:03
Configured IM Versions: [1L00, 1K00, 1J00, 1I00, 1H00, 1G00, 1F00, 1E00, 1D00, 1C00, 1B10, 1B00]

### 🦄 Related requirements

🦄 #xyz

**Comment by jordanpadams** [comment]
@jshughes is this something that seems reasonable based upon how the IM works? or should we add a flag for this?

**Comment by msbentley** [comment]
@jordanpadams @jshughes I should add that we only have this as a step in our dictionary build procedure because originally we understood this was needed for mapping of PDS4 to OpenSearch data types when ingesting data into the registry (otherwise the values end up as string?). As yet we haven't used this in anger, so I guess the questions are:

- is this still desired with the current registry?
- is this still the best way to get that mapping into the registry?
- if yes, does exporting the whole model and feeding this to the registry manager etc. cause conflicts 
- etc. etc.

**Comment by jordanpadams** [comment]
@msbentley 
> is this still desired with the current registry?
> is this still the best way to get that mapping into the registry?
Yes. The software uses these JSON files to determine the types of all attributes prior to loading it. This enables accurate search.

> if yes, does exporting the whole model and feeding this to the registry manager etc. cause conflicts

No. The only way this would be an issue would be if an attribute in the core IM changed datatype, which rarely happens, if ever. Even then, registry manager will just ignore it.

**Comment by jshughes** [comment]
@jordanpadams The "-J" option (export to JSON formatted file) has three possible outputs, but only two can be accommodated with a single flag. 

1) if LDDTool is run with the "-l" option, i.e. an LDD run, then 
		a) only the primary LDD namespace is exported to JSON
		   -or-
		b) all LDD namespaces in the run are exported to JSON
		
2) if not an LDD run, then only the Common namespace is exported to JSON
		
This ticket will change 1b to 1a.

Do we need a second option for 1b? My understanding was that the system uses the json output. Wouldn't
a "all inclusive" json file, similiar to the "all inclusive" data dictionary be useful?  
Suggestion "-j".

**Comment by jordanpadams** [comment]
@jshughes yes. let's add another flag for 1b, maybe `--include-all-namespaces` and be sure to include in the description that this is intended to be executed along with the `--json` flag. Could this also apply to the docbook export as well?

**Comment by jshughes** [comment]
> Could this also apply to the docbook export as well?

@jordanpadams Yes, this makes sense. I will put it in my queue.
